cypress/support/commands.ts

```diff
@@ -62,13 +62,7 @@ Cypress.Commands.add("login", () => {
 });
 
 /* preserveCookies */
-Cypress.Commands.add("preserveCookies", () => {
-  Cypress.Cookies.preserveOnce(
-    LOGIN_COOKIE,
-    "mci-session",
-    "mci-project-cookie"
-  );
-});
+Cypress.Commands.add("preserveCookies", () => {});
```

Comment:
likewise if its a no op we should remove it.

Reply:
If these functions are being removed, remember to also remove them from `e2e.ts` which is the corresponding file where they are declared